src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,10 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface ArtifactDependency {
+    val dependsOnBuildConfiguration: BuildConfiguration
+    val branch : String?
+    val pathRules: String
```

Comment:
The property name is plural but it returns a single String. So what will it return if there are several artifact rules? In TeamCity Web UI multiple rules are separate by a newline, but how this will be represented in this field? \n, \r\n, or it's somehow encoded? It would be better to return list of rules here so users of the API won't need to think about that.
Also it makes sense to parse each rule and return an object with separate source path, destination path, included/excluded options, otherwise each user of the API will need to write the same code which parses the rule string.